apps/storefront/src/components/B3ProductList.tsx

```diff
@@ -142,6 +145,7 @@ export default function B3ProductList<T>(props: ProductProps<T>) {
     textAlign = 'left',
     money,
     type,
+    getDigitalDownloadLinks = noop,
```

Comment:
Instead of adding a `noop` here, you can call the method conditonally: `getDigitalDownloadLinks?.()`